R/visr.R

```diff
@@ -268,13 +268,13 @@ visr.survfit <- function(
 
   gg <- ggplot2::ggplot(tidy_object, ggplot2::aes(x = time, group = strata)) +
     ggplot2::geom_step(ggplot2::aes(y = est, col = strata)) + 
-    ggplot2::scale_x_continuous(name = x_label,
-                                breaks = x_ticks,
+    ggplot2::scale_x_continuous(breaks = x_ticks,
                                 limits = c(min(x_ticks), max(x_ticks))) +
-    ggplot2::scale_y_continuous(name = y_label,
-                                breaks = y_ticks,
+    ggplot2::xlab(x_label) +
```

Comment:
is this for the theming? Would be helpful to put a rationale in your commit :)

Reply:
Oh sorry, I should've copied the problem from the linked issue :D